expected/13.8/extra/sqlite_fdw_post.out

```diff
@@ -4624,16 +4624,16 @@ DROP FUNCTION f_test(int);
 ALTER FOREIGN TABLE ft1 ALTER COLUMN c8 TYPE int;
 --Testcase 273:
 SELECT * FROM ft1 WHERE c1 = 1;
-ERROR:  invalid input syntax for type =1, column type =3
+ERROR:  SQLite data affinity "text" disallowed for PostgreSQL type "integer" = SQLite "integer", value = 'foo'
```

Comment:
Failed case: I think it should be `...disallowed for PostgreSQL data type ...` instead of `...disallowed for PostgreSQL type ...`

Reply:
Thank for your fixing.